templates/deployments.yaml

```diff
@@ -43,6 +43,24 @@ spec:
       {{- end }}
       # Sidecar container for generating .htpasswd with st2 username & password pair and sharing produced file with the main st2auth container
       initContainers:
+      - name: wait-for-db
+        image: busybox:1.28
+        command:
+          - 'sh'
+          - '-c'
+          - >
+            until nc -z -w 2 {{ .Release.Name }}-mongodb-headless 27017 && echo mongodb ok;
```

Comment:
We'll need to take the port number from the MongoDB chart. You can find some hints in st2.conf templating.

Same for RabbitMQ.

Reply:
Done